set.mm

```diff
@@ -469149,6 +469232,208 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Field Extensions
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c /FldExt /FinExt /AlgExt [:] $.
+
+  $( Syntax for the field extension relation. $)
+  cfldext $a class /FldExt $.
+
+  $( Syntax for the finite field extension relation. $)
+  cfinext $a class /FinExt $.
+
+  $( Syntax for the algebraic field extension relation. $)
+  calgext $a class /AlgExt $.
+
+  $( Syntax for the field extension degree operation. $)
+  cextdg $a class [:] $.
+
+  ${
+    $d e f $.
+    $( Definition of the field extension relation.  (Contributed by Thierry
+       Arnoux, 29-Jul-2023.) $)
+    df-fldext $a |- /FldExt = { <. e , f >. | ( ( e e. Field /\ f e. Field )
+      /\ ( f = ( e |`s ( Base ` f ) ) /\ ( Base ` f ) e. ( SubRing ` e ) ) ) }
+      $.
+
+    $( Definition of the field extension degree operation.  (Contributed by
+       Thierry Arnoux, 29-Jul-2023.) $)
+    df-extdg $a |- [:] = ( e e. _V , f e. ( /FldExt " { e } )
+      |-> ( dim ` ( ( subringAlg ` e ) ` ( Base ` f ) ) ) ) $.
+
+    $( Definition of the finite extension relation.  (Contributed by Thierry
+       Arnoux, 29-Jul-2023.) $)
+    df-finext $a |- /FinExt = { <. e , f >. | ( e /FldExt f
+      /\ ( e [:] f ) e. NN0 ) } $.
+
+    $d e f p x $.
+    $( Definition of the algebraic extension relation.  (Contributed by Thierry
+       Arnoux, 29-Jul-2023.) $)
+    df-algext $a |- /AlgExt = { <. e , f >. | ( e /FldExt f
+      /\ A. x e. ( Base ` e ) E. p e. ( Poly1 ` f )
+        ( ( eval1 ` p ) ` x ) = ( 0g ` e ) ) } $.
+  $}
+
+  ${
+    $d E e f $.  $d F e f $.
+    $( The field extension is a relation.  (Contributed by Thierry Arnoux,
+       29-Jul-2023.) $)
+    relfldext $p |- Rel /FldExt $=
+      ( ve vf cv cfield wcel wa cbs cress co csubrg cfldext df-fldext relopabiv
+      cfv wceq ) ACZDEBCZDEFQPQGNZHIORPJNEFFABKABLM $.
+
+    $( The field extension relation explicited.  (Contributed by Thierry
+       Arnoux, 29-Jul-2023.) $)
+    brfldext $p |- ( ( E e. Field /\ F e. Field ) -> ( E /FldExt F <-> ( F = (
```

Comment:
What does the prefix "br" mean? Binary Rrelation? I would propose to call this theorem simply "fldext".

Reply:
Yes, it stands for binary relation. I wanted to follow the same model as for example:
- `df-lmic` -> [`brlmic`](https://us.metamath.org/mpeuni/brlmic.html)
- `df-ric` -> [`brric`](https://us.metamath.org/mpeuni/brric.html)
- `df-gic` -> [`brgic`](https://us.metamath.org/mpeuni/brgic.html)
- `df-ssc` -> [`brssc`](https://us.metamath.org/mpeuni/brssc.html)
- `df-wdom` -> [`brwdom`](https://us.metamath.org/mpeuni/brwdom.html)
- `df-en` -> [`bren`](https://us.metamath.org/mpeuni/bren.html)
- `df-dom` -> [`brdom`](https://us.metamath.org/mpeuni/brdom.html)
- `df-tpos` -> [`brtpos`](https://us.metamath.org/mpeuni/brtpos.html)
- `df-pss` -> [`brtpos`](https://us.metamath.org/mpeuni/brtpos.html)

Reply:
I'll first push the change for the other comment, and wait for your opinion for the rest.

Reply:
OK, this seems to be a kind of convention I was not aware of.